addons/event/models/event_mail.py

```diff
@@ -36,7 +36,7 @@ def _selection_template_model(self):
     event_type_id = fields.Many2one(
         'event.type', string='Event Type',
         ondelete='cascade', required=True)
-    notification_type = fields.Selection([('mail', 'Mail')], string='Send', default='mail', required=True)
+    notification_type = fields.Selection([('mail', 'Mail')], compute="_compute_notification_type", string='Send', default='mail', required=True, readonly=True)
```

Comment:
`readonly=True`is the default for compute field without `inverse`

and for stability, this should be a `stored=True` field